uwblueprint/jump-math
# feat: Compute Top Performing Students

## Notion ticket link

[Compute Top Performing Students](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Implemented resolver, function and query to compute the top 5 students for a given testSessionId
<img width="1153" alt="Screenshot 2023-07-08 at 3 54 11 PM" src="https://github.com/uwblueprint/jump-math/assets/87676999/e4ac2937-46ce-4383-9053-36bc845361a7">



## Steps to test
1. Ensure existing tests are accurate
2. Call function on test sessions within the backend, (ensure expired date has passed)



## What should reviewers focus on?
* 


## Checklist
